en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md

```diff
@@ -0,0 +1,266 @@
+3. Click **Upload**.
+4. Select created Amazon Lambda Deployment Package (ZIP archive) and Upload.
+
+## Create Execution Role
+You need to create an Execution Role using [Amazon Lambda Connector Configuration](../amazonlambda-configuration.md) documentation.  
+
+## Deployment
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}
+
+## Testing
+1. Log in to the Micro Integrator CLI tool.
+
+    ```
+    ./mi remote login
+    ```
+2. Provide default credentials admin for both username and password.
+
+3. In order to view the proxy services deployed, execute the following command.
+
+    ```
+    ./mi api show
+    ```
+4. Send a POST request using a CURL command or sample client.
+  ``` 
+  curl -v POST -d 
+  '{
+    "secretAccessKey":"xxxx",
+    "accessKeyId":"xxxx",
+    "region":"us-east-2",
+    "blocking":"false",
+    "s3Bucket":"eiconnectortest",
+    "s3Key":"addingNumbers.zip",
+    "s3ObjectVersion":"null",
+    "functionName":"eiLambdaConnector",
+    "handler":"addingNumbers.addingNumbers",
+    "role":"arn:aws:iam::610968236798:role/EIConnectorTestRole",
+    "runtime":"python3.7",
+    "apiVersionCreateFunction":"2015-03-31"
+    }' "http://localhost:8290/createFunction" -H "Content-Type:application/json"  
+ ```
+5. See the following message content.
+
+ ```
+ {
+        "Description": "",
+        "TracingConfig": {
+            "Mode": "PassThrough"
+        },
+        "VpcConfig": null,
+        "RevisionId": "4b6e5fdd-cbfa-4ba2-9f6e-528cccdb333f",
+        "LastModified": "2020-03-13T05:33:54.900+0000",
+        "FunctionName": "eiLambdaConnector",
+        "Runtime": "python3.7",
+        "Version": "$LATEST",
+        "LastUpdateStatus": "Successful",
+        "Layers": null,
+        "FunctionArn": "arn:aws:lambda:us-east-2:610968236798:function:eiLambdaConnector",
+        "KMSKeyArn": null,
+        "MemorySize": 128,
+        "LastUpdateStatusReason": null,
+        "DeadLetterConfig": null,
+        "Timeout": 3,
+        "Handler": "addingNumbers.addingNumbers",
+        "CodeSha256": "VAISY9lY/a7DvxZNOSKCj+q/fsbfUaJjKhCsCVG3yzU=",
+        "Role": "arn:aws:iam::610968236798:role/EIConnectorTestRole",
+        "MasterArn": null,
+        "CodeSize": 405,
+        "State": "Active",
+        "StateReason": null,
+        "Environment": null,
+        "StateReasonCode": null,
+        "LastUpdateStatusReasonCode": null
+    }
+```
+6. Log in to the AWS Management Console.
+
+7. Navigate to the AWS Lambda and Functions tab.
+   <img src="/assets/img/connectors/AWSLambdaFunction.png" title="Amazon Lambda Function" width="800" alt="Amazon Lambda Function"/>
+   
+8. You can simply execute the function as follows.
+   
+   Navigate to **Configure test events**
+   <img src="/assets/img/connectors/configureTestEvent.png" title="Configure Test Event" width="800" alt="Configure Test Event"/>
+   
+   **Create Test Event**
```

Comment:
Step 10. Also better add how to create a Test Event.

Reply:
Updated.